### PR TITLE
Draft - Introduce standard user

### PR DESCRIPTION
**Pull request by sangeethah**
Changes to enable having both standard user and global admin users -  https://github.com/rancher/rancher/issues/23318

This will enable rbac cases to be implemented which would involve creating users and run all other tests as standard user who is a cluster owner.

### CODE REVIEW

File: tests/validation/tests/v3_api/common.py [hunk 1]
```diff
@@ -1289,15 +1300,36 @@ def validate_catalog_app(proj_client, app, external_id, answer=None):
...
+
+
+def get_user_token(user, cattle_auth_url=CATTLE_AUTH_URL):
+    r = requests.post(cattle_auth_url, json={
+        'username': user.username,
+        'password': user.username,
+        'responseType': 'json',
+    }, verify=False)
+    print(r.json())
+    return r.json()["token"]
```
Comment (soumyalj) [inline tests/validation/tests/v3_api/common.py]: password can be different than user name as a good practice.

**Comment by soumyalj** [comment]
LGTM for test_aks_cluster.py, test_bkp_restore.py, test_gke_cluster.py, test_import.py, test_project_quota.py, test_workload.py. One comment for common.py.

**Comment by jiaqiluo** [comment]
LGTM 
```
test_upgrade.py
test_sa.py
test_registry.py
test_rbac.py
test_nfs.py
test_network_policy.py
test_k8s_version_networkmodes.py
test_logging.py
test_custom_host_reg.py
```

One comment for `test_rke_cluster_provisoning.py`

**Comment by sangeethah** [comment]
@soumyalj Made sure that all users (including admin) is getting the password from env variable - USER_PASSWORD.

In test_custom_host_reg.py

```
    # Change admin password
    client = rancher.Client(url=RANCHER_SERVER_URL+"/v3",
                            token=token, verify=False)
    admin_user = client.list_user(username="admin").data
    admin_user[0].setpassword(newPassword=USER_PASSWORD)
```
In common.py

```
def create_user(client, cattle_auth_url=CATTLE_AUTH_URL):
    user_name = random_name()
    user = client.create_user(username=user_name,
                              password=USER_PASSWORD)
    client.create_global_role_binding(globalRoleId="user",
                                      subjectKind="User",
                                      userId=user.id)
    user_token = get_user_token(user, cattle_auth_url)
    return user, user_token
```

**Comment by soumyalj** [comment]
LGTM

File: tests/validation/tests/v3_api/test_rke_cluster_provisioning.py [hunk 2]
```diff
@@ -645,7 +645,18 @@ def create_and_vaildate_cluster(client, nodes,
     node_pools = []
     for node in nodes:
         node["clusterId"] = cluster.id
-        node_pool = client.create_node_pool(**node)
+        success = False
+        start = time.time()
+        while not success:
+            if time.time() - start > 10:
+                raise AssertionError(
+                    "Timed out waiting for cluster member global Roles")
```
[the remark this answers is not in the document]
Comment (sangeethah) [reply tests/validation/tests/v3_api/test_rke_cluster_provisioning.py]: It is relating to cluster owner global role.
Changed the the message.

**Comment by jiaqiluo** [comment]
LGTM

**Pull request merged**